src/leader/denominator.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Given an array A consisting of N integers, returns index of any
+ * element of array A in which the dominator of A occurs
+ *
+ * Detected time complexity: O(N*log(N)) or O(N)
+ * @param {number[]} A
+ */
+function solution(A) {
+  let count = 1;
+  let maxCount = count;
+  let item = A[0];
+
+  const B = JSON.parse(JSON.stringify(A));
```

Comment:
What is this trying to accomplish?

Reply:
@skaex, I needed to preserve the original indexes of the elements before sorting. This is to be used in the `return` statement